src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Layers/WfsXmlQuery/WfsXmlQuery.xaml.cs

```diff
@@ -0,0 +1,86 @@
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using Esri.ArcGISRuntime.Data;
+using Esri.ArcGISRuntime.Mapping;
+using System;
+using System.Diagnostics;
+using System.Windows;
+
+namespace ArcGISRuntime.WPF.Samples.WfsXmlQuery
+{
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        "Load WFS with XML query",
+        "Layers",
+        "Load a WFS feature table using an XML query.",
+        "")]
+    public partial class WfsXmlQuery
+    {
+        // Constants for the service URL, table name, and the query.
+        private const string XmlQuery = @"
+<wfs:GetFeature service=""WFS"" version=""2.0.0""
+  xmlns:Seattle_Downtown_Features=""https://dservices2.arcgis.com/ZQgQTuoyBrtmoGdP/arcgis/services/Seattle_Downtown_Features/WFSServer""
+  xmlns:wfs=""http://www.opengis.net/wfs/2.0""
+  xmlns:fes=""http://www.opengis.net/fes/2.0""
+  xmlns:gml=""http://www.opengis.net/gml/3.2"">
+  <wfs:Query typeNames=""Seattle_Downtown_Features:Trees"">
+    <fes:Filter>
+      <fes:PropertyIsEqualTo>
+        <fes:ValueReference>Trees:SCIENTIFIC</fes:ValueReference>
+        <fes:Literal>Tilia cordata</fes:Literal>
+      </fes:PropertyIsEqualTo>
+    </fes:Filter>
+  </wfs:Query>
+</wfs:GetFeature>
+";
+        private const string TableUrl = "https://dservices2.arcgis.com/ZQgQTuoyBrtmoGdP/arcgis/services/Seattle_Downtown_Features/WFSServer?service=wfs&request=getcapabilities";
+        private const string LayerName = "Seattle_Downtown_Features:Trees";
+
+        public WfsXmlQuery()
+        {
+            InitializeComponent();
+            Initialize();
+        }
+
+        private async void Initialize()
+        {
+            // Create the map with basemap.
+            MyMapView.Map = new Map(Basemap.CreateNavigationVector());
+
+            try
+            {
+                // Create the WFS feature table from URL and name.
+                WfsFeatureTable wfsTable = new WfsFeatureTable(new Uri(TableUrl), LayerName);
+
+                // Set the feature request mode.
+                wfsTable.FeatureRequestMode = FeatureRequestMode.ManualCache;
+
+                // Load the table.
+                await wfsTable.LoadAsync();
+
+                // Create a feature layer to visualize the table.
+                FeatureLayer statesLayer = new FeatureLayer(wfsTable);
+
+                // Add the layer to the map.
+                MyMapView.Map.OperationalLayers.Add(statesLayer);
+
+                // Populate the feature table with the XML query.
+                await wfsTable.PopulateFromServiceAsync(XmlQuery, true);
+
+                // Zoom to the extent of the query results.
+                await MyMapView.SetViewpointGeometryAsync(wfsTable.Extent, 50);
```

Comment:
RedFlag!
Will this line of code need to be updated/changed given the in-development status of 100.5? I see in build 2353 we now have a WfsFeatureTable.PopulateFromServiceWithXmlAsync(String, Boolean) method but not a WfsFeatureTable.PopulateFromServiceAsync(String,Boolean) method. Is this sample out of date with the current builds?

Reply:
There is a visual studio bug that causes a much older cached nuget package when switching between branches. In this case, the WFS branch is based on a version of v.next that didn't use PackageReference, and that seems to trigger the issue. 

The code is correct as it appears in the PR.